l10n_es_aeat_sii_oca/models/account_move.py

```diff
@@ -731,6 +731,10 @@ def _is_sii_simplified_invoice(self):
         is_simplified = partner.sii_simplified_invoice
         return is_simplified
 
+    def _get_sii_base_cost(self):
+        self.ensure_one()
+        return sum([x["base"] for x in self._get_tax_info().values()])
```

Comment:
No es redundante volver a obtener la información de impuestos? Debería estar ya hecho a otro nivel.

Reply:
@pedrobaeza Sí, ya me lo planteé.

Una alternativa es calcularlo a partir del `desglose_factura` que nos devuelve el `_get_sii_in_taxes`, algo del estilo `sum([x['BaseImponible'] for x in list(desglose_factura.values())[0]['DetalleIVA']])` y algo similar para las ventas. 

Otra alternativa és añadir un nuevo valor de retorno `total_base_imponible` a los métodos `_get_sii_in_taxes/_get_sii_out_taxes`. 

Pero en cualquiera de los dos casos, el código es poco leíble, implica tocar más partes del código y es muy dependiente de cualquier cambio hecho en la estructura de los diccionarios generados en los métodos `_get_sii_in_taxes/_get_sii_out_taxes`. 

Aún otra opción seria pasar como parámetro el resultado de _get_tax_info en la llamada a `_get_sii_in_taxes/_get_sii_out_taxes`, para así tenerlo disponible también dentro de los métodos `_get_sii_invoice_dict_in/_get_sii_invoice_dict_out`, quizás esta última es la que tiene menos problemas aunque implique cambiar los parámetros de los métodos.

¿Como lo ves?